Classes/Parser/ScssParser.php

```diff
@@ -45,6 +47,16 @@ protected function _compileFile($inputFilename, $preparedFilename, $outputFilena
         try {
             $this->parser->setVariables($this->overrides);
             $this->parser->setImportPaths(array(dirname($inputFilename), dirname($preparedFilename)));
+            if (ApplicationContext::isDevelopmentModeActive() || $this->config['enableDebugMode'] === '1') {
```

Comment:
from my POV i would rather say, dev context AND enableDebugMode, would also change the less parser accordingly - respecting the context feels more natural here.
https://github.com/kaystrobach/TYPO3.dyncss_less/blob/master/Classes/Parser/LessParser.php#L104

Reply:
whats your opinion?

Reply:
nope, please not. ;)

There are times when you need to debug "ad hoc", and I hate it when I have to login to the BE in order to adapt a setting. As EM config is serialized array it's really tough to change from CLI (Not sure wether typo3_console supports it?). 

Also it is really tough to keep ext_conf_template.txt settings different per environment, when you have everything in SCM and have deployments with adapted settings (.env, HostConfiguration) per site/installation.

So I am in big favor of having a simple mechanism to toggle this feature, without EM settings ;)

Reply:
ok, then i would opt for deprecating the debugMode and simply switch to contexts ...

Reply:
What's wrong with having two different options?

In general TYPO3 Context will take care of properly enabling/disabling the feature

If you want to use sourcemaps, but for some reason(production?) don't want to set development context, well then enableDebugMode.

I don't see the cost involved of keeping enableDebugMode while prefering Context.